### PR TITLE
[PR-17191] [Website] Add pages for docs.talkable.com

### PR DESCRIPTION
**Pull request by Georgiy-G**
## Demo

https://deploy-preview-260--talkable-docs.netlify.app/

## Related Stories

[](https://talkable.atlassian.net/browse/PR-17191)

### CODE REVIEW

**Comment by pustovalov** [comment]
@Georgiy-G `[PR-17191] [Website] Add pages for docs.talkable.com WIP` is it still WIP?

**Comment by Georgiy-G** [comment]
@pustovalov sorry, it was invalid name

File: source/payment_solutions/recharge.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  **Version 2**  of the integration also allows for accrued discounts in case the customer refers to multiple friends.
+
+5. If the Gleam widget is enabled, it can behave differently depending on the customer’s actions (if it’s set up according to a non-default flow):
+
+  Once the Advocate receives a reward, they will be shown a Gleam widget that will be showing the coupon code as well as
+  a notice that the coupon would be automatically applied to the ongoing subscription if the advocate has the one. The discount is still available for use on regular Shopify flow, but once used it won’t be applied to the subscription of the customer.
+
+**Contact us**
+
+Interested in setting this up? Contact your CSM or get in touch here "and have "here" link to email for support@talkable.com
```
Comment (pustovalov) [inline source/payment_solutions/recharge.rst:42]:
```suggestion
Interested in setting this up? Contact your CSM or get in touch here.
```

here - should be a link

File: source/email_marketing_and_automation/attentive.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  Thanks for your recent purchase! A referral from you would mean a lot to us! Share ‘Brand name’ with friends & we’ll share $10 with you 😃
+
+3. Trigger referral messages through SMS for increased engagement.
+
+  For example, send advocates their referral reward via text to increase redemptions and repeat shoppers.
+
+**Contact us**
+
+Interested in setting this up? Contact your CSM or get in touch here "and have "here" link to email for support@talkable.com
```
Comment (pustovalov) [inline source/email_marketing_and_automation/attentive.rst:38]: same here

**Review by pustovalov — APPROVED** [review]

**Pull request merged**